sources/packages/backend/workflow/test/2021-2022/parttime-assessment/awards-amounts/parttime-assessment-awards-amount-CSGP.e2e-spec.ts

```diff
@@ -0,0 +1,50 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CSGP.`, () => {
+  it("Should determine federalAwardCSGPAmount when awardEligibilityCSGP is true when studentDataApplicationPDPPDStatus is true", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+
+    // Assert
+    expect(calculatedAssessment.variables.programYearTotalCSGP).toBe(130);
+    expect(calculatedAssessment.variables.federalAwardNetCSGPAmount).toBe(3870);
+    expect(calculatedAssessment.variables.finalFederalAwardNetCSGPAmount).toBe(
```

Comment:
Can we check the expected in the same way the others are checked instead of checking against a Camunda variable?